cmd/egress-v4-cni-plugin/snat/snat.go

```diff
@@ -43,13 +43,17 @@ func iptRules4(target, src net.IP, chain, comment string, useRandomFully bool) [
 }
 
 // Snat4 SNATs IPv4 connections from `src` to `target`
-func Snat4(target, src net.IP, chain, comment string) error {
+func Snat4(target, src net.IP, chain, comment, randomizeSNAT string) error {
 	ipt, err := iptables.NewWithProtocol(iptables.ProtocolIPv4)
 	if err != nil {
 		return fmt.Errorf("failed to locate iptables: %v", err)
 	}
 
-	rules := iptRules4(target, src, chain, comment, ipt.HasRandomFully())
+	useRandomFully := true
+	if randomizeSNAT == "none" {
```

Comment:
1. shall we support for "--random" as well? i.e. randomizeSNAT == "hashrandom"
2. we should check ipt.HasRandomFully() as well, and if it's not supported, fallback to either "--random" or none. (depends on question 1)